core/Cargo.toml

```diff
@@ -16,6 +16,7 @@ ethcontract = "0.7.2"
 futures = { version = "0.3.5", features = ["compat"] }
 futures-timer = "3.0.2"
 isahc = { version = "0.9.6", features = ["json"] }
+jsonrpc-core = "14.2.0"
```

Comment:
This is not needed and re-exported through `ethcontract` as `use ethcontract::jsonrpc`. I would prefer to use the re-export to ensure that the jsonrpc-core version always matches the version needed by `ethcontract` and `web3`.